.circleci/config.yml

```diff
@@ -1,19 +1,20 @@
 unit_tests: &unit_tests
   steps:
     - checkout
-    - setup_remote_docker
     - restore_cache:
         key: dependency-cache-{{ checksum "package-lock.json" }}
     - run:
         name: NPM Rebuild
         command: npm install
+    - run:
+        name: Build project.
+        command: npm run build
```

Comment:
Require for https://github.com/webpack-contrib/uglifyjs-webpack-plugin/blob/master/src/uglify/index.js#L12